<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Coffee Journal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="coffeeTab">
       <attribute name="title">
        <string>Сорта</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="searchLayout">
          <item>
           <widget class="QLineEdit" name="coffeeSearchEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="coffeeSearchBtn">
            <property name="text">
             <string>Поиск</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="coffeeClearBtn">
            <property name="text">
             <string>Очистить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="buttonsLayout">
          <item>
           <widget class="QPushButton" name="addCoffeeBtn">
            <property name="text">
             <string>Добавить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editCoffeeBtn">
            <property name="text">
             <string>Редактировать</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteCoffeeBtn">
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="refreshCoffeeBtn">
            <property name="text">
             <string>Обновить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="coffeeTable"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="brewingTab">
       <attribute name="title">
        <string>Сессии</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="searchLayout2">
          <item>
           <widget class="QLineEdit" name="brewingSearchEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="brewingSearchBtn">
            <property name="text">
             <string>Поиск</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="brewingClearBtn">
            <property name="text">
             <string>Очистить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="buttonsLayout2">
          <item>
           <widget class="QPushButton" name="addBrewingBtn">
            <property name="text">
             <string>Добавить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="editBrewingBtn">
            <property name="text">
             <string>Редактировать</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteBrewingBtn">
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="refreshBrewingBtn">
            <property name="text">
             <string>Обновить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="brewingTable"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="statsTab">
       <attribute name="title">
        <string>Статистика</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTextEdit" name="statsText"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
